Makefile

```diff
@@ -43,8 +43,14 @@ test: build
 	$(GOTEST) -v ./... 2>&1
 
 integration-test: build
+	# temporary workaround, remove next line when x/net false positive is fixed
+	echo 'CVE-2018-17142\nCVE-2018-17846\nCVE-2018-17143\nCVE-2018-17847\nCVE-2018-17848' > packages/testdata/.nancy-ignore
```

Comment:
I would add a `CVN-111 until=2021-01-01` type thing to this, so they are only ever ignored for like a week or two. We really don't want to have these ignored forever if they become applicable.

Reply:
Yes! Done: e313db8  Also added comment to Issue #189